collections/profile.js

```diff
@@ -508,6 +522,7 @@ CollegeProfileSchema = new SimpleSchema({
   "parents": {type: _parentSchema, optional: true},
   "siblings": {type: _siblingSchema, optional: true},
   "highschool": {type: _highschoolSchema, optional: true},
+  "college": {type: _collegeSchema, optional: true},
```

Comment:
What does this field refer to?  "college" is a pretty non-specific name for this app.  Is it a current college?